sonar-orchestrator/src/main/java/com/sonar/orchestrator/db/DefaultDatabase.java

```diff
@@ -341,7 +341,31 @@ private DefaultDatabase dropDatabase(Connection connection) {
 
   private DefaultDatabase createDatabase(Connection connection) {
     LOG.info("Create database");
-    return executeDdl(connection, databaseClient.getCreateDdl());
+    executeDdl(connection, databaseClient.getCreateDdl());
+    if (databaseClient instanceof PostgreSql && databaseClient.getDBMajorVersion() >= 15) {
+        grantPermissionOnSchema();
+    }
+    return this;
+  }
+
+  /**
```

Comment:
@pierre-guillot-sonarsource I think we should move this method to Postgresql class in order to avoid this test "databaseClient instanceof PostgreSql && "